### PR TITLE
Add documentation about selection of chunk encoding

### PR DESCRIPTION
**Pull request by beorn7**
@fabxc @brian-brazil 

Note: This goes into the next-release branch.

### CODE REVIEW

File: content/docs/operating/storage.md [hunk 1]
```diff
@@ -59,6 +59,49 @@ not what you want for actual operations. The flag `storage.local.retention`
...
+a slightly different algorithm. A number of different encoding schemes are
+available for sample values. The choice is made per chunk based on the nature
+of the sample values (constant, integer, regularly increasing, random…). Major
+parts of the type 2 encoding are inspired by a paper published by Facebook
+engineers:
+[_Gorilla: A Fast, Scalable, In-Memory Time Series Database_](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf).
+
+With type 2, access within a chunk has to happen sequentially, and the encoding
+and decoding cost is a bit higher. Overall, type 2 will cause more CPU usage
+and increased query latency compared to type 1 but offers a once more much
```
Comment (brian-brazil) [inline content/docs/operating/storage.md]: s/once more //

**Comment by brian-brazil** [comment]
:+1:

File: content/docs/operating/storage.md (hunk 1 above)
Comment (beorn7) [inline content/docs/operating/storage.md]: I tried to express the fact that it is yet another improvement of the compression ratio, type 0 < type 1 < type 2.

Comment (brian-brazil) [inline content/docs/operating/storage.md]: That's two paragraphs away, so it ends up being confusing.

Comment (beorn7) [inline content/docs/operating/storage.md]: German brains have a larger stack size? I find it confusing the other way round... anyway, I trust your competence as a native speaker.

**Pull request merged**